machineconfiguration/v1/types.go

```diff
@@ -398,6 +398,24 @@ type MachineConfigPoolSpec struct {
 	// The targeted MachineConfig object for the machine config pool.
 	// +optional
 	Configuration MachineConfigPoolStatusConfiguration `json:"configuration"`
+
+	// pinnedImageSets is a list of PinnedImageSetRef objects that should be
+	// applied to the nodes in this pool.
```

Comment:
I believe this is 

```
// +patchStrategy=merge
// +patchMergeKey=name
```

Reply:
Oh I more meant from an end user perspective, what happens if I have multiple PISs specified, what happens to the image lists, is there a priority in which they are pulled, is there a merge strategy for the lists, or are they pulled in order, if there's a maximal list of images that the CRI can handle, what happens if the combination of PISs exceeds this.

I'm basically trying to think through various failure modes and what they may mean to a user

Reply:
Yeah I am updating the docs but basically, the controller will do a pre check to ensure that there is enough storage to pull the sets. If not it will set the pool degraded with an error message describing the problem.

So the CRI does not impose a limit to my knowledge it is limited by the resources availabl.e.